web/src/ApplicationEditPage.js

```diff
@@ -218,6 +218,14 @@ class ApplicationEditPage extends React.Component {
             }} />
           </Col>
         </Row>
+        <Row style={{marginTop: '20px'}} >
+          <Col style={{marginTop: '5px'}} span={(Setting.isMobile()) ? 22 : 2}>
+            {Setting.getLabel(i18next.t("provider:Jwt secret"), i18next.t("provider:Jwt secret - Tooltip"))} :
+          </Col>
+          <Col span={22} >
+            <Input value={this.state.application.jwtSecret} disabled />
+          </Col>
+        </Row>
         <Row style={{marginTop: '20px'}} >
```

Comment:
Either through editing `conf/app.conf` , or through the UI(such as xxxEdit Page), both appear is a bit redundant